docs/_articles/en/v3.x/troubleshooting.md

```diff
@@ -26,6 +26,32 @@ Please check if the affected Visualforce pages/components render correctly.
 If it does, check if it has an HTML tag that has an attribute with a dot? PMD has an [open issue](https://github.com/pmd/pmd/issues/2765) and we are working with them to fix it.
 If this is a new issue, please let us know.
 
+### The `scanner:run` command throws a warning about duplicate violations between ESLint and ESLint-LWC.
```

Comment:
Can we include the actual text from the warning somewhere so that a person searching with their string can find it immediately? Doesn't have to be in the question's title, but wherever you think would make sense.